internal/plugins/ansible/v1/scaffolds/init.go

```diff
@@ -35,15 +37,16 @@ import (
 	ansibleroles "github.com/operator-framework/operator-sdk/internal/plugins/ansible/v1/scaffolds/internal/templates/roles"
 
 	"github.com/operator-framework/operator-sdk/internal/kubebuilder/machinery"
+	"github.com/operator-framework/operator-sdk/internal/version"
 )
 
 const (
-	// KustomizeVersion is the kubernetes-sigs/kustomize version to be used in the project
-	KustomizeVersion = "v3.5.4"
```

Comment:
I'd prefer not change it from here. 
Reason. see that when a new release of kb be pushed we need to patch helm/ansible and it is easier identified if we keep the code as similar as possible. See; https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugin/v3/scaffolds/init.go#L38-L47

For example, if kb receives a patch to update the KustomizeVersion we will be able to identify it in a git diff which would make easier this process.

Reply:
Maybe we should not define this in the first place if we want to make sure we are using the same version as with KB?

Reply:
I'd revise this according to your suggestion anyway since it is not the focus of this PR.